content/language-agnostic/encyclopedia/code-editor.md

```diff
@@ -0,0 +1,18 @@
+<!--
+---
+Subjects:
+  - "What is a Code Editor?"
```

Comment:
We're trying to use "Subjects" in a slightly less intuitive sense. "Subjects" are categories already use to organize Codecademy content on the site, and so we're trying to use those same terms here whenever possible. I tried to explain a little better in the Editorial Standards for the Metadata section of [content-standards.md ](https://github.com/Codecademy/codepedia-content/blob/main/documentation/content-standards.md)

If you can find the right subjects from the list on Author that's linked to from those standards, please use those here. And if that list on Author doesn't seem to cover it, please just let us know in the channel--I suspect we're going to have to create an expanded list of Subjects for Codepedia.